cmd/admin-revoker/main.go

```diff
@@ -130,6 +130,21 @@ func revokeByReg(regID int64, reasonCode core.RevocationCode, deny bool, rac rpc
 	return
 }
 
+// This abstraction is needed so that we can use sort.Sort below
```

Comment:
Whoops, I missed the comment about moving this in the PR description, and did not expand enough of the diff to see where `sort.Sort` gets called. LGTM.